ow_lander/src/ow_lander/actions.py

```diff
@@ -1042,6 +1043,14 @@ def execute_action(self, goal):
 
     self._pub_trigger.publish()
 
+    # apply power usage for 2 cameras
+    ### FIXME: Exposure does not appear to be simulated in time (e.g. a 10
+    ###   second exposure will not take 10 seconds to complete), so this model
+    ###   per camera power consumption does not actually work the way it should.
```

Comment:
Were you intending to fix this for this PR, or as future work?

Reply:
Future work. I would not even know where to start having never worked on the stereo cam.